<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbOparation</class>
 <widget class="QDialog" name="DbOparation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>444</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>516</width>
    <height>444</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>516</width>
    <height>444</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Database Operation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>WhatsThisCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);</string>
       </property>
       <property name="text">
        <string>Database Opeartion Panel:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="toolTip">
        <string/>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="icon">
         <iconset resource="icons/rec.qrc">
          <normaloff>:/adddb.ico</normaloff>:/adddb.ico</iconset>
        </attribute>
        <attribute name="title">
         <string>Add DB</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>DB Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>148</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset resource="icons/rec.qrc">
               <normaloff>:/adddb.ico</normaloff>:/adddb.ico</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>169</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>[+] Status</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="icon">
         <iconset resource="icons/rec.qrc">
          <normaloff>:/loadd.ico</normaloff>:/loadd.ico</iconset>
        </attribute>
        <attribute name="title">
         <string>Load DB</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <pointsize>22</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Databases List</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidget_load"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLineEdit" name="lineEdit_loaddbPass">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Database Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>Load</string>
               </property>
               <property name="icon">
                <iconset resource="icons/rec.qrc">
                 <normaloff>:/dbload.ico</normaloff>:/dbload.ico</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>28</width>
                 <height>28</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="icon">
         <iconset resource="icons/rec.qrc">
          <normaloff>:/dbdelete.ico</normaloff>:/dbdelete.ico</iconset>
        </attribute>
        <attribute name="title">
         <string>Delete DB</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <pointsize>22</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Databases List</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidget_delete"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLineEdit" name="lineEdit_deletePass">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Db password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>Delete</string>
               </property>
               <property name="icon">
                <iconset resource="icons/rec.qrc">
                 <normaloff>:/remove.ico</normaloff>:/remove.ico</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>28</width>
                 <height>28</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="icon">
         <iconset resource="icons/rec.qrc">
          <normaloff>:/changepass.ico</normaloff>:/changepass.ico</iconset>
        </attribute>
        <attribute name="title">
         <string>Change Pass</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>22</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Databases List</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_change"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Old Password  </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_chOldPass">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>New Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_chNewPass">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>138</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>Change</string>
             </property>
             <property name="icon">
              <iconset resource="icons/rec.qrc">
               <normaloff>:/reload.ico</normaloff>:/reload.ico</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/rec.qrc"/>
 </resources>
 <connections/>
</ui>
